klib/src/platform/windows/posix.def

```diff
@@ -1,3 +1,4 @@
+depends = windows
```

Comment:
We usually do deps in another direction, see osx.def and linux.def. They depend on POSIX, not the opposite, so that common types are named same way across the platforms.

Reply:
Here posix.def includes winsock2.h, and winsock2.h includes windows.h
So all windows trash goes to posix.
Maybe it could be somewhat filterted, but it's not easy.

Reply:
Please wait until https://github.com/JetBrains/kotlin-native/pull/966 will get merged. I tried to make deps more reasonable, so your change could be made easy.